Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -102,9 +111,13 @@ struct HomeView: View {
                     and: survey
                 )
             }
-            .highPriorityGesture(DragGesture().onEnded {
-                handleSwipe(translation: $0.translation.width)
-            })
+            .gesture(
```

Comment:
I'm not sure why we need to add this gesture as the original gesture works fine. Adding this makes the original gesture seems broken as not using the same transition. 

https://user-images.githubusercontent.com/6356137/202122170-44a10366-51ed-48b8-bccf-f6ad45a005a2.mp4

Reply:
@blyscuit Nice catch 🙌 .

```swift
.highPriorityGesture(DragGesture().onEnded {
                handleSwipe(translation: $0.translation.width)
            })
```
This gesture works well for paging tabView, but it's somehow disables vertical scroll gesture 😢 . Because of that I applied the new gesture to both use scroll and swipe gestures. I just decreased DragGesture's minimumDistance to `11.0` at https://github.com/nkhanh44/ic-survey-ios/pull/74/commits/27e264fe0c3417605de8447f04b5c5c01a43914e, it reproduced less error swipe than the old one.
I still don't have the better solution for this conflict. Do you have any idea? Thank you 🙏 .

Reply:
Could you explain what's the issue when we don't have this new gesture? I briefly try without it and don't see any issue with the original gesture.

Reply:
@blyscuit The issue with the original gesture is that I couldn't scroll the view like pulling to refresh as the video below 😄.

https://user-images.githubusercontent.com/25881847/202159980-12996219-0b6c-4538-98a4-4d54ca49e3b3.mp4

Reply:
That's weird I don't have this issue when I commented out the new gesture 🤔 

https://user-images.githubusercontent.com/6356137/202162171-a2e1b9d0-057c-4a2d-b627-e353d3993c70.mp4

Reply:
Can we disable the gesture on the tab view instead and keep this new gesture?

Reply:
Morning @blyscuit 🙋,
 When Implementing the Home screen I wanted to create the swiping behavior like [the Figma](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=38%3A2083&scaling=scale-down), blink blink behavior, I added the new gesture for it. And then when implementing this PR, that gesture blocks the scrolling, I decided to add another gesture but it seems not work well with swiping😄 .
To solve this issue, I think we can remove the gesture safely without breaking any [IC's requirement](https://www.notion.so/nimblehq/Internal-Certification-Path-Mobile-de354e1ce76f4b80922c3db21ea0117f). Without the new gesture, it works well with the same transition as you mentioned, 🙇 .
What do you think? 🙏 

**Figma:**
https://user-images.githubusercontent.com/25881847/202351863-1ede2da7-2a3e-49fc-a70d-6647d0bcdd62.mov

**Without gesture:**

https://user-images.githubusercontent.com/25881847/202352881-64e15cd5-27d8-4da1-9775-085989351bc3.mp4

Reply:
@nkhanh44 This looks good to me, if you want the fade animation instead of the sliding animation, maybe you have to disable the tab bar scroll or remove tab bar and use other component instead. You can try playing around with that in a new issue. This one can close 👍

Reply:
Thank you so much for the feedback 🙌 .